container/split.go

```diff
@@ -185,9 +185,12 @@ func (r *splitContainerRenderer) computeSplitLengths(total, lMin, tMin float32)
 
 // Declare conformity with interfaces
 var _ fyne.CanvasObject = (*divider)(nil)
-var _ fyne.Draggable = (*divider)(nil)
-var _ desktop.Cursorable = (*divider)(nil)
-var _ desktop.Hoverable = (*divider)(nil)
+
+var (
+	_ fyne.Draggable     = (*divider)(nil)
+	_ desktop.Cursorable = (*divider)(nil)
+	_ desktop.Hoverable  = (*divider)(nil)
+)
```

Comment:
This is strange. It seems to me like these should have been in one block.

Reply:
agreed. bug?

Reply:
Yup. I’ve opened an issue upstream about it https://github.com/mvdan/gofumpt/issues/212.

Reply:
I’ve done this change manually for now. It doesn’t complain without the fix to the automatic formatting so we are probably fine for now.